Include/refcount.h

```diff
@@ -372,6 +385,36 @@ static inline void Py_DECREF(PyObject *op)
 #define Py_DECREF(op) Py_DECREF(_PyObject_CAST(op))
 
 #elif defined(Py_REF_DEBUG)
+static inline void Py_DECREF_MORTAL(const char *filename, int lineno, PyObject *op)
```

Comment:
```suggestion
static inline void decref_mortal(const char *filename, int lineno, PyObject *op)
```

(This doesn't seem to follow our naming conventions.)

Reply:
Almost all of our `static inline` functions start with `Py` and many are capitalized, probably because they were macros once.
This is adding "mortal" to `static inline void Py_DECREF`, and `Py_DECREF_mortal` looks weird.

If it were a non-inline `static` function the `decref_mortal` would be best.